sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1116,6 +1118,267 @@ def max_deflection(self):
         else:
             return None
 
+    @doctest_depends_on(modules=('matplotlib',))
```

Comment:
There is no need to use `doctest_depend_on` after [#14984](https://github.com/sympy/sympy/pull/14984).